tests/CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ set(
     0078-c_from_cpp.cpp
     0079-fork.c
     0081-fetch_max_bytes.cpp
+    0083-io_event_cb.c
```

Comment:
this file will need to be added to win32/tests/tests.vcxproj too

Reply:
Will do. I'll also rename the test to cb_event, according to the naming convention.